openshift/templates/agent/agent-deploy.yaml

```diff
@@ -421,7 +421,7 @@ parameters:
     displayName: Set read only ledger flag.
     description: Enable/Disable the read only ledger flag.  Controls the inclusion/exclusion of the agent '--read-only-ledger' startup parameter.  Defaults to disabled.
     required: true
-    value: "true"
+    value: "false"
```

Comment:
It doesn't look like this parameter was added to the param files: we should add it and update it there, rather than in the main deployment configuration, to allow each agent to be deployed in read only mode or not independently.

Reply:
AGENT_READ_ONLY_LEDGER is in all the agent param files.. maybe you're looking at a different commit?